opentera-webrtc-native-client/OpenteraWebrtcNativeClient/python/test/socketio_inactive_client_test.py

```diff
@@ -15,16 +15,9 @@ def tearDownClass(cls):
         cls._signaling_server_runner.close()
         super(SocketIOInactiveClientTestCase, cls).tearDownClass()
 
-    def setUp(self):
-        super(SocketIOInactiveClientTestCase, self).setUp()
-
-    def tearDown(self):
-        super(SocketIOInactiveClientTestCase, self).tearDown()
-
```

Comment:
@mamaheux Should these be completely removed?

Reply:
Right, because the ones in the base class will be called anyway du to inheritance